src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
 
 Once executed, this query will return a single Workflow named _Create OpenStack Instance_.
 
+=== Send Workflows to another Catalog Bucket, from another Scheduler or the local one.
+
+In the Workflow Catalog, it is possible to select Workflows from the selected bucket and send them to another bucket. This bucket can belong to the same Catalog or to another Scheduler's Catalog.
+
+==== Send Workflows directly
+
+If the Workflows to share are sent to a Scheduler having a public API, or to another bucket of the local Scheduler, it is convenient to send them directly.
+
+Select the bucket then the Workflows. Choose `Send to another scheduler` and click `OK`.
+
+==== Export Workflows into an archive, then Import them
+
+If the remote Scheduler is running in another network, of if you cannot use direct sending for some reason, you can export Workflows into an archive, that you can transfer by any file transfer method (email, usb key, etc.) and then import this archive into the other Scheduler.
```

Comment:
"in another network" is strange to me, but I don't know how to change it. 

of if you cannot ... --> or the direct sending can not be used for some reason, 

that you can transfer --> which can be transferred ... (), and then import the archive to the Scheduler you want to use.